resources/sdk/clisdkclient/extensions/transform_data/transform_data.go

```diff
@@ -42,8 +48,26 @@ func convertArrayOfObjectsToMap(data string) []map[string]interface{} {
 
 func ProcessTemplateFile(mp interface{}) string {
 	path := []string{TemplateFile}
-	name := filepath.Base(path[0])
-	tmpl := handleParse(template.New(name).Funcs(sprig.TxtFuncMap()).ParseFiles(path...))
+	fileName = filepath.Base(path[0])
+
+	if strings.Contains(TemplateFile, "github") {
```

Comment:
See if you can make a regex that matches the accepted URL patterns. At the moment you could cod it with something like github.gotmpl